protocol/storage/client.go

```diff
@@ -301,7 +302,7 @@ func (smc *Client) isMaybeDupDeal(p *storagedeal.Proposal) bool {
 
 // LoadVouchersForDeal loads vouchers from disk for a given deal
 func (smc *Client) LoadVouchersForDeal(dealCid cid.Cid) ([]*types.PaymentVoucher, error) {
-	storageDeal := smc.api.DealGet(dealCid)
+	storageDeal := smc.api.DealGet(context.TODO(), dealCid)
```

Comment:
`Client#LoadVouchersForDeal` should accept a context. `API#Payments` should pass its context to this method.